### PR TITLE
Update tokens.json

### PR DESCRIPTION
**Pull request by achinta3**
## How to add token?

- [x] Add metadata to [`tokens.json`](tokens.json)

```json
{
    "name": "Rewalty",
    "logo": "https://raw.githubusercontent.com/achinta3/LOGO/main/COIN.png",
    "logo_lg": "https://raw.githubusercontent.com/achinta3/LOGO/main/COIN.png",
    "symbol": "REWA",
    "account": "wepevt3petjr",
    "chain": "eos"
}
```

- [x] Add token logo `*.png` to [`./logos`](./logos) folder
- [x] Update `README.md` with **Symbol** & **Contract Name**

### CODE REVIEW

**Comment by achinta3** [comment]
Please Add token information.

**Review by DenisCarriere — CHANGES_REQUESTED** [review]
the COIN.png file is empty

File: tokens.json [hunk 1]
```diff
@@ -7,6 +7,14 @@
         "account": "eosyieldeosy",
         "chain": "eos"
     },
+    {
+        "name": "Rewalty",
+        "logo": "https://raw.githubusercontent.com/achinta3/LOGO/main/COIN.png",
+        "logo_lg": "https://raw.githubusercontent.com/achinta3/LOGO/main/COIN.png",
```
Comment (DenisCarriere) [inline tokens.json:13]: Must provide internal GitHub repo URL

**Review by DenisCarriere — CHANGES_REQUESTED** [review]

File: tokens.json (hunk 1 above)
Comment (achinta3) [reply tokens.json:13]: ok

**Pull request closed without merging**